portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -41,9 +46,14 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Get response from the form
     String text = request.getParameter("text-input");
+    long timestamp = System.currentTimeMillis();
```

Comment:
No need to change anything now, but something to think about:
Using `System.currentTimeMillis()` will make this class difficult to test. Let's chat about this offline (and perhaps the TDD walkthroughs will address this as well).

Reply:
ohh yes I remember this point from the unit testing lecture! I can bring this up in my 1:1 today to discuss